### DISCUSSION TITLE
Visualization regions Glasser atlas

### DISCUSSION THREAD

**Discussion by shuertaspenen** [simnibs/simnibs#178] [9. Results and analysis]
Hi, 

I have calculated the maximum E-field for a tDCS and obtained the index of this value (the triangle in the middle of gray matter). What I want to do now, is to mark it by using an arrow, point or something in the head plot I have with the E-fields. Is this possible? 
I also know to which Glasser region this index is part of. Could I plot this in someway? 

Thank you :)

**Comment by axthi** [comment]
Hi, 
you can write the positions out in .geo-files, and overlay them with the mesh using file->merge in gmsh. Below is a code snippet to get started.

from simnibs import mesh_io
mesh_io.write_geo_spheres([[100, 0 ,0], [110, 0, 0]], 
						'mygeo.geo', values=[1,2], name='myspheres')
mesh_io.write_geo_vectors([[100, 0 ,0], [110, 0, 0]], 
						[[10, 0 ,0], [0, 10, 0]],
						'mygeo.geo', name='myvectors', mode='ba')
